src/tab/tab.svelte

```diff
@@ -0,0 +1,34 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import rippleEffect from '../_utils/ripple.js';
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+  export let inputClass = null;
+  export let contentClass = null;
+
+  export let value;
+  export let group = null;
+  export let disabled = false;
+  export let ripple = true;
+
+  const dispatch = createEventDispatcher();
+</script>
+
+<label class={classes('tab', _class)}>
+  <input
+    bind:group
+    {value}
+    type="radio"
+    class={inputClass}
+    {disabled}
```

Comment:
Isn't it redundant to include it as a prop when it can just be passed in `$$restProps` since it is not used anywhere else? (unlike `value`, which is used elsewhere.)

Reply:
I suppose I just wanted to document this prop to let people know that this is the way to disable tabs

Reply:
Documenting can be done in the documentation. I personally don't check out the source code unless I meet some weird behavior I need to explain

Reply:
Yeah, but it doesn't feel right to document a prop that isn't actually there.